R/crunchy-server.R

```diff
@@ -0,0 +1,101 @@
+#' A Shiny server with Crunch auth
+#'
+#' To make sure that users who access your shiny.crunch.io app are allowed to
+#' access it, use `crunchyServer()` instead of [shiny::shinyServer()], and wrap
+#' your UI content inside [crunchyBody()]. This will prevent anyone who is not
+#' logged into Crunch in their browser from accessing your app.
+#'
+#' To restrict access further to only certain Crunch users, you can set an
+#' authorization method, either by passing a server function to
+#' the `authz` argument of this function, or by calling
+#' [setCrunchyAuthorization()].
+#'
+#' @param func A `function (input, output, session)`, as you'd normally give to
+#' `shinyServer()`. If the user is not authenticated or authorized, this
+#' function will not be evaluated.
+#' @param authz A `function (input, output, session)` to evaluate
+#' to determine if the current user is authorized to enter the app. Ideally,
+#' this is cheap to execute because it will be called repeatedly.
```

Comment:
It took me a second to parse the logic here maybe change "Ideally, this is cheap to execute" to "Ideally, this should be cheap to execute" Or possibly flip the dependent clause "Since it will be called repeatedly, this should be cheap to execute"

Reply:
Thanks, I switched it to use the words you suggest.